tests/vendor/README.md

```diff
@@ -0,0 +1,5 @@
+Since Tecknicom/TCPDF works all the way back to PHP 5.3, it is not easy to have
```

Comment:
I'd rather not include any third party software for licensing and legal reasons.
Instead we could just disable the tests for the PHP versions that are not supported by this.

Reply:
Sure, I'll look into it.

Reply:
Fixed.